components/x-topic-search/src/AllFollowed.jsx

```diff
@@ -0,0 +1,29 @@
+import { h } from '@financial-times/x-engine';
+import styles from './TopicSearch.scss';
+import classNames from 'classnames';
+
+// transform like this => topic1, topic2 and topic3
+const transformFollowedTopics = followedTopicsIncludeSearchTerm => {
```

Comment:
I think this function can be named more better.

Something like `arrayToSentence` would be more descriptive and agnostic about what the subject matter was?